pyvrp/cpp/TimeWindowSegment.h

```diff
@@ -61,7 +64,7 @@ TimeWindowSegment TimeWindowSegment::merge(Matrix<int> const &dist,
 }
 
 template <typename... Args>
-TimeWindowSegment TimeWindowSegment::merge(Matrix<int> const &dist,
+TimeWindowSegment TimeWindowSegment::merge(Matrix<distance_type> const &dist,
```

Comment:
Should this be the duration matrix?

Reply:
I agree, maybe it was like that since there is no separate distance/duration yet? (Did I implement it? Don't remember)

Reply:
I don't know what caused it; I just noticed it when reading through the PR!